plugins/kubernetes-backend/api-report.md

```diff
@@ -278,6 +281,11 @@ export interface RouterOptions {
   logger: Logger;
 }
 
+// Warning: (ae-missing-release-tag) "runPeriodically" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export function runPeriodically(fn: () => any, delayMs: number): () => void;
```

Comment:
This shouldn't be added to the public interface.

Reply:
I thought it should so that custom implementations of `KubernetesClustersSupplier` can use it?  Or should they also copy-paste it like I did here from https://github.com/backstage/backstage/blob/master/plugins/search-backend-node/src/runPeriodically.ts already?

Reply:
Yeah we're intentionally removing this from public surfaces for now (did that elsewhere before too)

Reply:
Okay, I removed the export again and added a comment in the two examples that says to use a copy of `runPeriodically`.